ocspresponder.go

```diff
@@ -15,7 +18,63 @@ type CertDbSource struct {
 	Accessor certdb.Accessor
 }
 
-func NewSource(dbAccessor certdb.Accessor) cfocsp.Source {
+const interval = 96 * time.Hour
```

Comment:
If the interval is configured to be 96h, then should the background job also run every 96 hours?